ballerina/build.gradle

```diff
@@ -33,46 +40,68 @@ def stripBallerinaExtensionVersion(String extVersion) {
             return extVersion
         }
     } else {
-        return extVersion.replace("${project.ext.snapshotVersion}", '')
+        return extVersion.replace("${project.ext.snapshotVersion}", "")
     }
 }
 
+ballerina {
+    packageOrganization = packageOrg
+    module = packageName
+    testCoverageParam = "--code-coverage --coverage-format=xml"
+    buildOnDockerImage = "nightly"
```

Comment:
Isn't it required to change to the relevant Ballerina build version? This is what will reflect on Central after a publish.

Reply:
As per the offline discussion I had with @ThisaruGuruge, we need to have some changes from our Ballerina gradle plugin side to address this. IMO we can keep this as is, until we finalize it.